src/qililab/qprogram/qblox_compiler.py

```diff
@@ -103,10 +103,11 @@ def __init__(self):
         }
 
         self._qprogram: QProgram
+        self._bus_mapping: dict[str, str] | None
         self._buses: dict[str, BusCompilationInfo]
         self._sync_counter: int
 
-    def compile(self, qprogram: QProgram) -> dict[str, QPy.Sequence]:
+    def compile(self, qprogram: QProgram, bus_mapping: dict[str, str] | None = None) -> dict[str, QPy.Sequence]:
```

Comment:
bus_mapping argument missing in docstring

Reply:
Done